openedx/core/djangoapps/course_roles/permissions.py

```diff
@@ -0,0 +1,143 @@
+"""
+Permissions for course roles app.
+"""
+from enum import Enum, unique
+
+from django.utils.translation import gettext as _
+
+
+@unique
+class CourseRolesPermission(Enum):
+    """
+    Enum of all user permissions, the values are the permissions names
+    in the course_roles_permission table in database.
+    """
```

Comment:
Nit: What you have here is fine, but you could also make a `PermissionsData` attrs class that holds the human-readable short name and description, and make this an enumeration of those classes. So instantiation here would look like:

```
@unique
class CourseRolesPermissions(Enum):
    manage_content = PermissionData(
        _("Manage Content"),
        _("Can view, create, edit, delete and publish (not publisher tool) any course content."),
    )
    # etc...
```